packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -96,39 +145,29 @@ You should also ensure that the `useDefineForClassFields` setting is `false`. No
 
 Enabling `emitDecoratorMetadata` is not required and not recommended.
 
-### Using decorators with Babel  { #decorators-babel }
+### Using decorators with Babel { #decorators-babel }
 
-If you're compiling JavaScript with [Babel](https://babeljs.io/docs/en/), you can enable decorators by adding the following plugins and settings:
+[Babel](https://babeljs.io/docs/en/) supports standard decorators with the [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators) plugin. Babel does not support TypeScript experimental decorators, so you must use Lit decorators in standard decorator mode with the `accessor` keyword on decorated class fields.
```

Comment:
Link back to standard decorators section?
```suggestion
[Babel](https://babeljs.io/docs/en/) supports standard decorators with the [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators) plugin. Babel does not support TypeScript experimental decorators, so you must use Lit decorators in [standard decorator mode with the `accessor` keyword](#standard-decorators) on decorated class fields.
```

Reply:
Done!